src/sections/2021/section-4.js

```diff
@@ -128,6 +133,10 @@ const Section4 = () => (
         </BarChart>
       </Figure>
     </GridCell>
+
+    <GridCell align="center" startMD="8" startLG="12">
+        <ArtElement type="bars" />
+      </GridCell>
```

Comment:
I think that the indentation here may be off just a little.

Reply:
Ah good catch